{
  try {
    return doIsReady();
  }
 catch (  Exception ignored) {
    LOGGER.trace("Exception while checking if ready",ignored);
    return false;
  }
}
